app/controllers/schedules_controller.rb

```diff
@@ -12,7 +12,17 @@ def index
                                   "%#{params[:search_by_name]}%")
   end
 
-  def show; end
+  def show
+    Time.zone = 'America/Chicago'
```

Comment:
Do we want to hard code the time zone here?

Reply:
I thought about it..but it's a very TAMU specific app, so why not

Reply:
What if the client is on a vacation?

Reply:
:') I hope not
I will figure out how to use a dynamic time zone with my next feature